vhdbuilder/packer/init-variables.sh

```diff
@@ -119,7 +125,7 @@ if [[ "$MODE" == "linuxVhdMode" || "$MODE" == "windowsVhdMode" ]]; then
 	id=$(az sig show --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME}) || id=""
 	if [ -z "$id" ]; then
 		echo "Creating gallery ${SIG_GALLERY_NAME} in the resource group ${AZURE_RESOURCE_GROUP_NAME} location ${AZURE_LOCATION}"
-		az sig create --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME} --location ${AZURE_LOCATION}
+		az sig create --resource-group ${AZURE_RESOURCE_GROUP_NAME} --gallery-name ${SIG_GALLERY_NAME} --location ${AZURE_LOCATION} ${GALLERY_PERMISSIONS_COMMAND} ${}
```

Comment:
I don't think this will do what you want. this will not make the final image a community gallery image. this is merely the intermediate SIG that packer builds to.

you need to change the publishing logic in RP, which re-hydrates our prod SIG from a blob. there's a lot of weird steps today...

Reply:
yeah... problem is that our actual SIG publish happens in the weekly release pipeline rather than during the build here

current flow is

AgentBaker/packer VHD build -> intermediate SIG -> intermediate blob (<something>.vhd) -> another intermediate blob (prefetch optimized) -> (for AGC do blob copy to correct tenants/subs) -> managed image -> final prod SIG